packages/astro/src/core/errors/errors-data.ts

```diff
@@ -590,6 +633,13 @@ See https://docs.astro.build/en/guides/server-side-rendering/ for more informati
 	 * @name CLI Errors
 	 */
 	// CLI Errors - 8xxx
+	/**
+	 * @docs
+	 * @description
+	 * Astro encountered an unknown error while starting one of its CLI command. The error message should contain more information.
```

Comment:
I believe that when we use "one of" we need to use a plural noun instead.
```suggestion
	 * Astro encountered an unknown error while starting one of its CLI commands. The error message should contain more information.
```